### ISSUE TITLE
Write climate matching for bul bul

### ISSUE THREAD

**Issue by SanderDevisscher** [inbo/riparias-prep#20]
- [ ] Calculate no records per climate + %overlap

for these records:
[Uploading Pycnonotus_jocosus_records.zip…]()

@timadriaens I think this is best suited to be made into a function with these functionalities:
- [ ] ability to select file
- [ ] ability to select list of species

and with this as output:
- [ ] csv containing no records per climate zone + %overlap
- [ ] shp??

what do you think ?

also whats the deadline ?

**Comment by timadriaens** [comment]
@SanderDevisscher yes, would be so nice, we could use that workflow for future matching exercises and PRA's. Perhaps one functionality to add is the production of one or a few maps (for current/future climate): 

- [ ] map of the world with KG climate classification as underground and the occurrence data plotted on it
- [ ] global map of climatic suitability for the species based on %overlap)

Deadline for the pra is 20 May, but never mind if you have little time to stick to it, this is an 'extra' to the pra. If it works, we could run it for more species subject to PRA and see if the outcome corroborates the conclusions of the species distributions models.

This workflow, as well as the one for the RIPARIAS climate matching, would be good to consolidate as a package/function, the same way it is done for TrIAS. I think there is great scope for it to be used worldwide by the invasion community, for example for horizon scanning, drafting alert lists, risk assessment etc. 

Therefore: could you include Damiano in the discussion/revision of code and turning it into a package/function?

**Comment by SanderDevisscher** [comment]
@damianooldoni I'll write a function for climate matching. You can later integrate the code into the RIPARIAS - package. 

ok for you ?.

**Comment by damianooldoni** [comment]
If needed (see #21), yes.

**Comment by timadriaens** [comment]
this function is best integrated in the TrIAS package, so moved [here](https://github.com/trias-project/trias/issues/73)

**Comment by SanderDevisscher** [comment]
@timadriaens I close this issue since the function has been written for the Trias package see [#77](https://github.com/trias-project/trias/pull/77)

**Comment by SanderDevisscher** [comment]
After integration into the Trias package (allmost) @timadriaens  still needs the cm for Pycnonotus jocosus (2486151) & Pycnonotus cafer (2486131)

**Comment by SanderDevisscher** [comment]
`20_cm_bul_bul` - branch
